src/dom.js

```diff
@@ -98,11 +98,19 @@ export const _DOM = {
       }
     }
   },
-  injectStyles(styles: string, destination?: Element, prepend?: boolean): Node {
+  injectStyles(styles: string, destination?: Element, prepend?: boolean, id?: string): Node {
     let node = document.createElement('style');
     node.innerHTML = styles;
     node.type = 'text/css';
 
+    if (id) {
+        let oldStyle = document.getElementById(id);
+        if (oldStyle) {
```

Comment:
I feel that we should also verify that the element found by id is actually a style element before removing it.

Reply:
@ZHollingshead 
@HamedFathi and I are friends, so I try to answer on behalf of him 😄 

> I feel that we should also verify that the element found by id is actually a style element before removing it.
this sounds rational, hence it will prevent developers from accidentally removing another element, personally, I prefer to throw an error rather than doing nothing

Reply:
I would agree with that.

Reply:
@shahabganji Thanks for your answers.

@ZHollingshead thanks for your feedback :) In my opinion, the review of tag ID is a good idea so I updated.